src/components/ArtistList/ArtistList.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Artist from '../components/Artist/Artist';
+import { Link } from 'react-router-dom';
+
+const ArtistList = ({ artists }) => {
+  const artistList = artists.map((artistsObject) => (
+    <Link key={ artistsObject.id } to='/'>
```

Comment:
So now that your artists are all showing up you need to get each link to match the paths you stated in app.js. So this should look something like this — ` <Link key={artistsObject.id} to={`/artist/${artistsObject.id}`}>
      <Artist name={artistsObject.name}/>
    </Link>`